app/services/get-sub-nav.js

```diff
@@ -34,7 +34,7 @@ module.exports = function (id, organisationalUnitName, currentPath, workloadType
         navigation.push(new Link('NPS Caseload', baseLink + '/caseload'))
         navigation.push(new Link('CRC Caseload', baseLink + '/crc-caseload'))
         navigation.push(new Link('Case Progress', baseLink + '/case-progress'))
-        if (authorisation === false || userRole === 'Data Admin' || userRole === 'System Admin' || userRole === 'Manager') {
+        if (authorisation === false || userRole === 'Super User' || userRole === 'System Admin' || userRole === 'Manager') {
```

Comment:
ugh! both authorisation and userRole are passed into this method purely to write this horrible if statement! We have got a right job on our hands here. 
I appreciate the imperative not to rewrite while test coverage is not there and the scaffolding is so weak.
As a small first step towards improving it perhaps we could use the constants from app/constants/user-roles.js ?